src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/UploadFileItem.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-list-item>
+    <v-row class="d-flex align-center">
+      <!-- File Name and Size -->
+      <v-col cols="4">
+        <div>{{ file.name }} ({{ (file.size / 1024).toFixed(2) }} KB)</div>
+      </v-col>
+
+      <!-- Progress Bar or Upload Completed -->
+      <v-col cols="4">
+        <div v-if="isUploadComplete">Upload completed</div>
+        <v-progress-linear v-else :buffer-value="uploadProgress" height="20" color="primary"></v-progress-linear>
+      </v-col>
+
+      <!-- Delete Button -->
+      <v-col cols="4" class="d-flex justify-end">
+        <v-icon @click="deleteFile">mdi-delete</v-icon>
```

Comment:
the figma designs want this symbol 
![image](https://github.com/bcgov/ECC-ECER/assets/74216496/7fc98534-95ba-4521-81e5-3584c80b973c)

can change this to mdi-trash-can-outline

they also want a tooltip with red color when you hover 
![image](https://github.com/bcgov/ECC-ECER/assets/74216496/7cc5ecf0-90fd-455e-a4e1-1e773449d26f)

tool tip seems alright. Not sure about the red hover color. If it's an easy win sure?

Reply:
Added tooltip, not doing red colour... Our other v-btn + v-tooltip combo's do not do a hover colour.